pmgr/pmgrobj.py

```diff
@@ -230,6 +227,15 @@ def __init__(self, table, hutch, debug=False, prod=True):
         self.checkForUpdate()
         self.updateTables()
 
+    def connect(self, prod):
+        d = {}
+        with open(CREDENTIALS % ("prod" if prod else "dev")) as f:
```

Comment:
It'd be nice to use the standard library instead of rolling our own config file format here.
configparser is pretty easy to use and actually simplifies this:
https://docs.python.org/3/library/configparser.html

```python
import configparser

conf = configparser.ConfigParser(defaults={"default_setting": "def"})
conf.read("pmgr.ini")
print(conf["PRODUCTION"]["user"])
print(conf["PRODUCTION"]["default_setting"])
print(dict(conf["PRODUCTION"]))
```

results in
```
prod-user
def
{'host': 'prod-host', 'user': 'prod-user', 'password': 'prod-password', 'db': 'prod-db', 'default_setting': 'def'}
```

for this file:
```
[PRODUCTION]
host=prod-host
user=prod-user
password=prod-password
db=prod-db

[DEVELOPMENT]
host=dev-host
user=dev-user
password=dev-password
db=dev-db
```

Those headers `[]` aren't optional, just so you know, so the config file format would have to change slightly to include them. If you _really_ want separate files for these for some reason, each could just have `[DEFAULT]` as their section (per the docs)

Reply:
Done.